src/app/supplies/page.tsx

```diff
@@ -45,18 +55,52 @@ const MUITable = () => {
     setFilteredRows(rows);
   };
 
-  const fetchAvailableSupplyData = async () => {
+  const fetchSupplyPointsData = async () => {
     try {
       const response = await apiClient.get("/supplies", {
-        withCredentials: true
+        headers: {
+          "Content-Type": "application/json",
+        },
+        data: {}, // Añadir cuerpo vacío para solucionar error de "Content-Type vacío"
```

Comment:
What is the error that is preventing this empty body?

Reply:
I got an error when sending the request that the Content-Type field is required. If you specify it, then it is necessary to add a body (in this case, an empty body). I found an explanation here (https://es.stackoverflow.com/questions/381254/agregar-content-type-a-axios)